### PR TITLE
br_dp2p_port

### PR DESCRIPTION
**Pull request by mahdipourziaei-binary**

### CODE REVIEW

File: src/javascript/app/pages/cashier/dp2p.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        const is_svg = Client.get('landing_company_shortcode') === 'svg';
+        if (is_svg) {
+            import('@deriv/p2p').then((module) => {
+                const el_dp2p_container = document.getElementById('binary-dp2p');
+                const shadowed_el_dp2p = el_dp2p_container.attachShadow({ mode: 'closed' });
+
+                const el_main_css = document.createElement('style');
+                el_main_css.innerHTML = `
+                @import url(${urlForStatic('css/p2p.min.css')});
+                :host {
```
Comment (mustofa-binary) [inline src/javascript/app/pages/cashier/dp2p.js]: can we put it inside scss instead?

File: src/sass/static/pages.scss [hunk 2]
```diff
@@ -1609,3 +1609,65 @@ body #not_authenticated_financial ul.checked > li {
...
+    //font-family: $FONT_STACK;
+    //text-rendering: optimizeLegibility;
+    //-webkit-font-smoothing: antialiased;
+    //-moz-osx-font-smoothing: grayscale;
+    //-webkit-text-size-adjust: 100%;
+    //-ms-text-size-adjust: 100%;
+    //box-sizing: border-box;
+    position: relative;
+    //
+    //h1, h2, h3, h4, h5, h6, p {
```
Comment (mustofa-binary) [inline src/sass/static/pages.scss]: we can remove the comments if its not being used

File: src/templates/app/cashier/dp2p.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// eslint-disable-next-line arrow-body-style
+const DP2P = () => {
+
```
Comment (mustofa-binary) [inline src/templates/app/cashier/dp2p.jsx]: nit: can remoce line here

File: src/javascript/app/pages/cashier/dp2p.js (hunk 1 above)
Comment (mahdipourziaei-binary) [reply src/javascript/app/pages/cashier/dp2p.js]: @mustofa-binary We cannot 👍 The styles should be placed inside the shadow DOM which we access through JS, and we need to get the path of the CSS file with `urlForStatic` function.

Comment (mustofa-binary) [reply src/javascript/app/pages/cashier/dp2p.js]: can you make a comment on it then, since its an unusual flow on BS

Comment (mahdipourziaei-binary) [reply src/javascript/app/pages/cashier/dp2p.js]: Sure I'll add it @mustofa-binary

**Pull request merged**